src/System.Memory/tests/Span/Clear.cs

```diff
@@ -192,19 +192,30 @@ public static void ClearValueTypeWithReferences()
             Assert.Equal<TestValueTypeWithReference>(expected, actual);
         }
 
-        [ActiveIssue(16492)]
-        [OuterLoop]
         [Fact]
-        public unsafe static void ClearLongerThanUintMaxValueBytes()
+        [OuterLoop]
+        public unsafe static void ClearTests()
+        {
+            // These tests need to be run serially with no risk of them running in parallel
+            // or the sum of the allocations will run the test machine out of memory and result
+            // in failures.
+            // Also, the native version should come before the managed version since the memory
+            // allocation and free are explicit instead of relying on the GC, so we should be
+            // certain memory was freed from the first before the second is called.
+            ClearNativeLongerThanUintMaxValueBytes();
```

Comment:
@jkotas I'm thinking more of the scenario where we allocate the 8 GB managed array, use it, test completes and exits. Next test is an 8 GB native allocation. What if GC has not collected yet and the 8 GB is still not released back to the system.

Should I force a garbage collection at the end of that managed array test?